spec/features/user_signs_up_spec.rb

```diff
@@ -5,7 +5,7 @@
   let!(:user) { FactoryGirl.create(:user) }
 
   it "with valid email and password" do
-    login("person1@example.com", "password")
+    login('person1@example.com', 'password')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.